examples/rvc-app/rvc-common/src/rvc-service-area-delegate.cpp

```diff
@@ -17,6 +17,7 @@
  */
 #include <app-common/zap-generated/attributes/Accessors.h>
 #include <rvc-service-area-delegate.h>
+#include <vector>
```

Comment:
vector is disallowed in core code because it grows quite aggressively and can eat and fragment memory fairly easily.

This isn't core code, so I'm not sure what the policy is here. Are you sure this is safe for all the implementations using this code?

@andy31415 - do we have code style policy here?

Reply:
@cecille right, this is just an example application so there are no concerns wrt to using this STL container here

Reply:
generally examples can have anything so can platform-specific files (e.g. linux, android and darwin have lots of resources). Common code in src/app/clusters or similar should not use heap.